### PR TITLE
Plotting and table joining 

### PR DESCRIPTION
**Pull request by infotroph**

### CODE REVIEW

**Comment by infotroph** [comment]
@dlebauer will work more on #3 tomorrow, but here's at least a start on plotting

File: traits/01-simulated-sorghum.Rmd [hunk 1]
```diff
@@ -157,14 +150,13 @@ library(traits)
...
-public_key <- 9999999999999999999999999999999999999999
+public_key <- "9999999999999999999999999999999999999999"
 
 traits <- c("Vcmax", "c2n_leaf", "cuticular_cond", "SLA", "quantum_efficiency", "leaf_respiration_rate_m2", "stomatal_slope.BB", "Jmax", "chi_leaf", "extinction_coefficient_diffuse")
 
 
-SLA <- betydb_search(query = "SLA", betyurl = terraref_test_url, key = public_key)
-%>% 
-  filter(scientificname == 'Sorghum bicolor')
+SLA <- (betydb_search(query = "SLA", betyurl = terraref_test_url, key = public_key)
```
Comment (dlebauer) [inline traits/01-simulated-sorghum.Rmd]: line 158 returns null; 159 throws error

File: traits/01-simulated-sorghum.Rmd [hunk 2]
```diff
@@ -183,15 +175,6 @@ ggplot(SLA) +
 
 
 traits <- betydb_search("SLA Vcmax",  betyurl = terraref_test_url, key = public_key)
```
Comment (dlebauer) [inline traits/01-simulated-sorghum.Rmd]: didn't like the pairs plot?

File: traits/04-agronomic-metadata.Rmd [hunk 3]
```diff
@@ -73,41 +73,43 @@ Here are some key tables and fields that we will look at:
...
+#%>% addMarkers(data = sites_point) # points removed by only querying Season 2
 
-citations <- (betydb_query(table="citations")
+citations <- (betydb_query(table = "citations", betyurl = betyurl, key = betykey, api_version = "beta")
   %>% select(citation_id = id, author, year, title))
 
-# Fails because terraref BETY's yields table is empty
-# Succeeds if betyurl="https://www.betydb.org/" (with a valid key...)
-yields = (betydb_query(table="yields")
+traits <- (betydb_query(table = "traits", betyurl = betyurl, key = betykey, api_version = "beta")
```
Comment (infotroph) [inline traits/04-agronomic-metadata.Rmd]: Traits table is empty too! ...at least when queried with the public key. @dlebauer is all the yield and trait data restricted to internal collaborators?

**Comment by infotroph** [comment]
@dlebauer I think this is now ready to merge except the "minor" problem that the joins in 04-agronomic-metadata.Rmd fail because there are no public-facing data in either the traits or the yields table. 

Not sure how to proceed there -- Show joining on a different set of tables? Use betydb.org instead of terraref?

**Pull request merged**